instructure/canvas-android
# [MBL-15489][Student][Teacher] Add the ability to add alt text to images in mobile applications

Test plan:
- Verify that when we add an image to our content in RCE there is a dialog that lets the user add alt text. Ideally this should be verified in all the screens with RCE.
- The easiest way to verify the alt text is to open the content in browser and inspect the element.
- If the dialog is cancelled or dismissed the image should be added, but without an alt text.
- Student use cases:
  - Create discussion, announcement, page in a group.
  - Submit text submission.
  - Reply to a discussion.
- Teacher use cases:
  - Create or edit discussion, announcement, page.
  - Edit syllabus.
  - Edit assignment details.
  - Reply to a discussion.

refs: MBL-15489
affects: Student, Teacher
release note: Added option to add alt text to images when editing or creating content.
